backend/copr_backend/background_worker_build.py

```diff
@@ -255,6 +255,7 @@ def _check_vm(self):
         # generalize it into a separate package that we could eventually use
         # here.
         cmd = "copr-builder-ready " + self.job.chroot
+        self.log.info("Running remote command: %s", cmd)
```

Comment:
nit: ton't you want to show stderr, if provided? might give us some warnings in advance

Reply:
We do

```
if rc:
    self.log.info(stderr)
```

which should IMHO work fine.

Reply:
but if rc == 0 and there's some output, we just swallow it

Reply:
I did it this way because for some reason my stderr is polluted by this line when testing this locally

```
Shared connection to builder closed.
```

So I intentionally wanted to swallow it. But we haven't seen that message in STG so I am confused.

Reply:
:shrug: if you want to hear my opinion, I don't think the logs need to be pretty - but must be complete (minus credentials, haha). But it is up to you, I already approved this PR.